packages/neos-ui/src/Containers/LeftSideBar/NodeTree/Node/index.js

```diff
@@ -1,9 +1,9 @@
 import React, {PureComponent} from 'react';
 import PropTypes from 'prop-types';
 import {$get} from 'plow-js';
-import {compose} from 'redux';
 import {connect} from 'react-redux';
 
+import compose from 'lodash.compose';
```

Comment:
In V4, `compose` is removed in favor of `flowRight`. Maybe use that right away, to save us from a breaking change in the future? https://github.com/lodash/lodash/wiki/Changelog
They are the same thing, just aliases.

Reply:
Or do we already use `lodash.compose` somewhere else? In this case it's ok.